IFC4x3/Properties/i/Increasing_34FH6g00XEcAJrdjKZ3gOO/Documentation.md

```diff
@@ -0,0 +1,5 @@
+The Increasing attribute is an optional attribute of the Pset_Stationing property set. It is meant for use in referents of predefined type REFERENCEMARKER being nested to a given alignment.
+
+If present and true, or if not present, then the relevant subsequently nested STATION referents are expected to have Pset_Stationing.Station values being larger than the REFERENCEMARKER's Pset_Stationing.Station value.
```

Comment:
If present and true, or if not present, then the relevant subsequently nested STATION referents are generally expected to have Pset_Stationing.Station values being larger than the REFERENCEMARKER's Pset_Stationing.Station value.

Reply:
Added "generally" to the text .